extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/AgroalRecorder.java

```diff
@@ -1,34 +1,27 @@
 package io.quarkus.agroal.runtime;
 
 import io.quarkus.arc.Arc;
-import io.quarkus.arc.runtime.BeanContainer;
-import io.quarkus.arc.runtime.BeanContainerListener;
+import io.quarkus.datasource.runtime.DataSourcesBuildTimeConfig;
+import io.quarkus.datasource.runtime.DataSourcesRuntimeConfig;
+import io.quarkus.datasource.runtime.LegacyDataSourcesRuntimeConfig;
 import io.quarkus.runtime.annotations.Recorder;
 
 @Recorder
 public class AgroalRecorder {
 
     public static final String DEFAULT_DATASOURCE_NAME = "<default>";
```

Comment:
This is now longer used anymore as far as I can tell

Reply:
Right, removed.

Reply:
Thanks